apps/emqx/etc/emqx.conf

```diff
@@ -1023,15 +1023,31 @@ broker {
   ## Dispatch strategy for shared subscription
   ##
   ## @doc broker.shared_subscription_strategy
-  ## ValueType: random | round_robin | sticky | hash
+  ## ValueType: random | round_robin | sticky | hash | local
   ##   - random: dispatch the message to a random selected subscriber
   ##   - round_robin: select the subscribers in a round-robin manner
+  ##   - local: select the local subscriber otherwise random cluster-wise
```

Comment:
This explanation leaves some questions, for example, what if there are multiple local subscribers.